lombok.config

```diff
@@ -1,3 +1,880 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
+##
```

Comment:
Is this file generated automatically? Seems we have much more content now

Reply:
This one is not. Nowl `ombok` deprecated auto-generate config. This is the config template that they provide.